src/components/Exchange/ConfirmationBox.js

```diff
@@ -149,6 +178,10 @@ export default function ConfirmationBox(props) {
   };
 
   const getPrimaryText = () => {
+    if (decreaseOrdersThatWillBeExecuted.length > 0 && !isTriggerWarningAccepted) {
+      return `Accept confirmation of trigger orders`;
+    }
```

Comment:
both `getPrimaryText` and `isPrimaryEnabled` use `getError()`. so it should be sufficient to add that check to `getError` func end return error text. so this works for any kind of error